src/main/java/thaumicboots/api/ItemBoots.java

```diff
@@ -282,7 +282,7 @@ public static void renderHUDJumpNotification() {
         Minecraft mc = Minecraft.getMinecraft();
         String text = getModeText(
                 "thaumicboots.jumpEffect",
-                getBoots(mc.thePlayer).stackTagCompound.getDouble(TAG_MODE_JUMP) * 100);
+                IBoots.getBoots(mc.thePlayer).stackTagCompound.getDouble(TAG_MODE_JUMP) * 100);
```

Comment:
switch back to getBoots(mc.thePlayer).stackTagCompound.getDouble(TAG_MODE_JUMP) * 100);

it'll be fine

Reply:
My bad about forgetting the ItemBoots class.

But DO NOT do this: https://github.com/GTNewHorizons/ThaumicBoots/pull/27#discussion_r1573985756

It will crash with EMT which does not use ItemBoots. It will cause an NPE.

Reply:
fixed locally about to push